core/src/components/menu/test/basic/menu.e2e.ts

```diff
@@ -74,6 +74,53 @@ test.describe('menu: basic', () => {
 
     await expect(scrollTop).toBe(200);
   });
+
+  test('should render on the correct side when side is changed dynamically', async ({ page }) => {
```

Comment:
Could we also add `skip.mode('md')` or `skip.mode('ios')` to both tests? These checks aren't mode-dependent, and the more screenshots we can trim down, the better. You can also just manually delete the screenshots for the mode you skipped.